lib/rubycritic/analysers_runner.rb

```diff
@@ -23,8 +23,8 @@ def initialize(paths)
       @paths = paths
     end
 
-    def run
-      ANALYSERS.each { |analyser| analyser.new(analysed_modules).run }
+    def run(analysers = nil)
```

Comment:
or this could be `opts = nil` and `(opts[:analysers] || ANALYSERS)` below? One of these versions seemed cleaner than first version, you can compare in commit history.